cmake/ValhallaPkgConfig.cmake

```
@@ -33,6 +33,9 @@ function(configure_valhalla_pc)
   if(ENABLE_SERVICES)
     list(APPEND REQUIRES libprime_server)
   endif()
+  if(ENABLE_GDAL)
+    list(APPEND REQUIRES GDAL::GDAL)
```

this need to be 
```suggestion
    list(APPEND REQUIRES gdal)
```

it's the name of the .pc file in <lib_dir>/pkgconfig/.